### ISSUE TITLE
NS-USBLoader detects VPN IP address - Not local IP (Nord Windows Client)

### ISSUE THREAD

**Issue by Azurren** [developersu/ns-usbloader#118]
First off please close and discard if you feel this user-error isn't worth fixing, thankyou for all your work on this project!
**Describe the bug (mandatory)**
NS-USBLoader picks up the IP address from the Nord VPN Windows client as opposed to the local IP address

**Information about environment (mandatory):**
 - OS: Windows 11
 - Java version: 1.8.0

**Steps to reproduce**
1. Connect to any VPN using the NordVPN windows client
2. Attempt to use NS-USBLoader over network
3. Send installation files to the Switch running Awoo
4. Confirm the Selection in Awoo and proceed with the installation
5. Awoo times out

**Expected behaviour**
Is it possible to differentiate between VPN connections / adapters and local connections in Windows?
[ PASS ] NET: Host IP detected as: 10.x.x.x.x
[ PASS ] NET: Your port detected as: 6721
[ INFO ] 	Start chain
[ PASS ] Initiation files list has been sent to NS.

**Did you see this issue in previous versions?**
Yes

**Additional context**
Thanks again for all your hard work! I spent way longer than I should have diagnosing this issue assuming it was firewall / switch version related. Hopefully this issue report will at least point other users in the right direction.

**Comment by developersu** [comment]
First of all, thank you for reporting this =)

Mmm.. I don't know if it's possible or not. 10.x.x.x looks like your local address.. at least it's from intra-net subnet range. Anyway, you can find the right one by executing `ipconfig` in Windows Terminal. There should be all your local IPs. As a workaround you can just set it manually in application settings and not rely on 'autodetection' function. 
Also it's a nice way to make sure that it's not a firewall issue.

**Comment by Azurren** [comment]
Yeah, the 10.x.x.x address is a local IP assigned by Nords virtual network adapter, locally I use a 192.168.0.0/24 range.
Thanks for pointing out that there is an option in the application settings for manually selecting the hosts IP address! That'll save me from having to disconnect my VPN connection in the future.

Thanks again!
